docs/instrumentation-list.yaml

```diff
@@ -708,13 +732,27 @@ libraries:
       - org.jboss.logmanager:jboss-logmanager:[1.1.0.GA,)
   jdbc:
   - name: jdbc
+    description: |
+      Traces JDBC operations such as statements, batches, commits, and rollbacks by injecting byte‑code into JDBC drivers and common connection‑pool proxies. Each call produces a span named after the SQL verb, enriched with standard DB client attributes (system, database, operation, sanitized statement, peer address) and error details if an exception occurs.
```

Comment:
While the javaagent use bytcode modification the library instrumentation does not. Currently no spans are created for commits and rollback executed through jdbc api methods (I guess a span would be created when you executed commit or rollback statements). I'm not sure whether `common connection‑pool proxies` should be mentioned as it is an implementation detail of the connection pools, for those not familiar with it this might be confusing.